BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -132,6 +137,21 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
         
     }
     
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        
+        if segue.identifier == "mapViewSegue" {
+            
+            if let destination = segue.destination as? StoresMapViewController {
```

Comment:
You probably don't need the condition above that checks the segue identifier. This condition check alone is enough.